pykickstart/commands/ostreecontainer.py

```diff
@@ -82,4 +85,9 @@ def _getParser(self):
     def parse(self, args):
         ns = self.op.parse_args(args=args, lineno=self.lineno)
         self.set_to_self(ns)
+
+        if self.handler.ostreesetup.seen:
```

Comment:
Shouldn't it also conflict with `liveimg`? And with `url` and `nfs` and `cdrom` and `harddrive` and `hmc`?
I just wonder if this is the best approach to solve conflicting commands in general.

Reply:
oops, put my comment in the main thread, sorry. I think that's valid, but should be handled in a new PR.